### DISCUSSION TITLE
keyboard arrows as keyboard response

### DISCUSSION THREAD

**Discussion by fspinar** [jspsych/jsPsych#2554] [Q&A] [answered]
Hey all, 

I am trying to use keyboard arrow buttons as inputs to canvas-keyboard-response plugin, but I cannot find how to represent them as strings. In other words:

var stimulus_procedure = {
  type: 'canvas-keyboard-response',
  canvas_size: [windowHeight, windowWidth],
   choices: ['f','j'],
  stimulus: ...
 }
 
 works fine, but what should I type for f & j in the case of keyboard arrows ? 
 
 Thanks !

**Comment by jodeleeuw** [comment, marked as answer]
Hi @fspinar 

`'ArrowLeft'` and `'ArrowRight'` are the strings for the left and right keys. You can use a site like [this one](https://keycode.info/) to find the string for a key.

**Reply by fspinar** [reply]
Thanks @jodeleeuw ! It worked
